packages/webservice/app/build.gradle.kts

```diff
@@ -16,9 +16,10 @@ repositories {
 dependencies {
     implementation(project(":models"))
     implementation("org.springframework.boot:spring-boot-starter-security")
-    implementation("com.yahoo.elide", "elide-spring-boot-starter", "5.0.0-pr12")
+    implementation("com.yahoo.elide", "elide-spring-boot-starter", "5.0.0-pr15")
     implementation("org.jetbrains.kotlin:kotlin-reflect")
     implementation("com.h2database", "h2", "1.3.176")
+    implementation( "org.hibernate", "hibernate-validator", "6.1.5.Final")
```

Comment:
is this dependency needed? The changes in the PR don't seem to be making use of validators.

Reply:
Without this, none of the existing validators actually work.